docs/fundamentals/jcloud/basic.md

```diff
@@ -143,16 +143,27 @@ export JCLOUD_NO_INTERACTIVE=1
 
 ## Get status
 
+To get the status of a Flow:
 ```bash
-jc status 173503c192
+jc status 15937a10bd
 ```
 
 ```{figure} status.png
 :width: 70%
 ```
 
+## Monitoring
+If the Flow has enabled [monitoring](https://docs.jina.ai/fundamentals/flow/monitoring-flow/) and deployed to JCloud, basic monitoring function will be provided. 
```

Comment:
```suggestion
To enable monitoring with the Flow, you can set `monitoring: true` in the Flow yaml and you'd be given access to a [Grafana](https://grafana.com/) dashboard.
```

Also, let's not add a hardcoded link here, instead use a reference. e.g.- [here](https://github.com/jina-ai/jina/blob/monitoring-doc/docs/fundamentals/jcloud/advanced.md#retention-days).